src/main/java/ch/njol/skript/expressions/ExprCommandInfo.java

```diff
@@ -159,6 +163,31 @@ public String toString(@Nullable Event event, boolean debug) {
 			(commandName == null ? "" : " of command " + commandName.toString(event, debug));
 	}
 
+	@Nullable
+	private Command[] getCommands(Event event) {
+		if (event instanceof ScriptCommandEvent && commandName == null)
+			return new Command[] {((ScriptCommandEvent) event).getScriptCommand().getBukkitCommand()};
+
+		CommandMap map = Commands.getCommandMap();
+		if (map == null)
+			return null;
+
+		if (commandName != null)
+			return commandName.stream(event).map(map::getCommand).filter(Objects::nonNull).toArray(Command[]::new);
+
+		String commandName;
+		if (event instanceof ServerCommandEvent) {
+			commandName = ((ServerCommandEvent) event).getCommand();
+		} else if (event instanceof PlayerCommandPreprocessEvent) {
+			commandName = ((PlayerCommandPreprocessEvent) event).getMessage().substring(1);
+		} else {
+			return null;
+		}
+		commandName = commandName.split(":")[0];
+		Command command = map.getCommand(commandName);
+		return command != null ? new Command[] {command} : null;
```

Comment:
These will never be reached due to L183

Reply:
What do you mean? L184 is only reached if the previous 2 conditions weren't met, aka the event wasn't `ServerCommandEvent` or a `PlayerCommandEvent`

Reply:
there is an else section that returns null

Reply:
Yes... But the other conditions don't return, so if one of them passes then the rest of the code is ran

Reply:
Pardon me, I should never review when I am sleepy, just got what you mean 😁